.gitignore

```
@@ -445,3 +445,4 @@ dmypy.json
 
 *.pickle
 **/imgs/*.pdf
+.vscode/settings.json
```

```suggestion
.vscode/
```

aqui talvez colocar só .vscode?